src/test/java/edu/harvard/iq/dataverse/pidproviders/PidUtilTest.java

```diff
@@ -114,8 +114,20 @@
 @JvmSetting(key = JvmSettings.HANDLENET_KEY_PASSPHRASE, value = "passphrase", varArgs ="hdl1")
 @JvmSetting(key = JvmSettings.HANDLENET_KEY_PATH, value = "/tmp/cred", varArgs ="hdl1")
 
+// CrossRef 1
+@JvmSetting(key = JvmSettings.PID_PROVIDER_LABEL, value = "CrossRef 1", varArgs = "crossref1")
+@JvmSetting(key = JvmSettings.PID_PROVIDER_TYPE, value = CrossRefDOIPidProvider.TYPE, varArgs = "crossref1")
+@JvmSetting(key = JvmSettings.PID_PROVIDER_AUTHORITY, value = "10.5072", varArgs = "crossref1")
+@JvmSetting(key = JvmSettings.PID_PROVIDER_SHOULDER, value = "FK2/", varArgs = "crossref1")
```

Comment:
These are very close to the values for the provider ez1 above - perhaps changing to 10.5075 and/or to FK5/ to make a few more characters different would make it clearer.

Reply:
will use sample CrossRef values